gitgud/__main__.py

```diff
@@ -185,13 +187,17 @@ def handle_progress(self, args):
 
         next_challenge = challenge.next_challenge
         if next_challenge is not None:
-            next_challenge.setup(self.file_operator)
-            self.file_operator.write_challenge(next_challenge)
+            self.handle_load_challenge(args, next_challenge)
         else:
             print("Wow! You've complete every challenge, congratulations!")
             print("If you want to keep learning git, why not try contributing to git-gud by forking us at https://github.com/bthayer2365/git-gud/")
             print("We're always looking for contributions and are more than happy to accept both pull requests and suggestions!")
 
+    def handle_load_challenge(self, args, challenge):
```

Comment:
`handle_*` methods should be reserved for the subcommands, so we can just rename this to `load_challenge`. Since it's not a handler anymore, it doesn't have to accept `args` anymore.  and have it accept just one argument, `challenge`.